src/qml/FeatureForm.qml

```diff
@@ -671,7 +671,7 @@ Page {
         height: 48
         clip: true
         bgcolor: form.state === 'Add' ? "#900000" : Theme.darkGray
-        visible: !qfieldSettings.autoSave || dontSave
+        visible: !dontSave
```

Comment:
It has been visible before when autosave off or dontsave. Why it's visible now only when it's not dontsave? Why not always true... Haven't looked deeper but looks odd to me...

Reply:
@signedav , I actually thought there was a logic flaw here, because dontSave is false by default & therefore would hide the buttons.

Reply:
Hm... ok. Maybe needs a deeper look.